plugins/contracts/src/generators/generateNamespace.ts

```diff
@@ -37,7 +37,9 @@ function generateNamespace(generatorNamespace: GeneratorNamespace, context: Cont
     generateNamespace(generatorNamespace, childContext),
   )
 
-  const statements = [...interfaceStatements, ...enumStatements, ...namespaceStatements]
+  const statements = [...interfaceStatements, ...enumStatements, ...namespaceStatements].filter(
+    statement => statement !== undefined,
+  )
```

Comment:
Hmm. Since `interfaceStatements` and `namespaceStatements` are `flatMap`s (so I assume they return `[]` when skipped). Then that means that only `enumStatements` can be `T | undefined`. I'd either switch `enumStatements` to also be flat map (and `generateEnum` to return an array) or make that filter directly only on `enumStatements`'s `map`